lookout/style/typos/templates/quality_on_identifiers.md.jinja2

```diff
@@ -0,0 +1,32 @@
+# Typos correction quality on identifiers
+{% do test.__setitem__(Columns.Split, test["wrong"].apply(tokenize)) %}
+{% do test.__setitem__(Columns.CorrectSplit, test["correct"].apply(tokenize)) %}
+{% do test.__setitem__(IDENTIFIER_INDEX_COLUMN, range(test|length)) %}
+{% set flat_test = flatten_df_by_column(test, Columns.Split, Columns.Token, str.split) %}
+{% set flat_correct = flatten_df_by_column(test, Columns.CorrectSplit, Columns.CorrectToken, str.split) %}
+{% do flat_test.__setitem__(Columns.CorrectToken, flat_correct[Columns.CorrectToken]) %}
+{% set flat_suggestions = {} %}
+{% for i, row in flat_test.iterrows() %}
+        {% do flat_suggestions.__setitem__(i, suggestions.get(row[IDENTIFIER_INDEX_COLUMN], {}).get(
+        row[Columns.Token], [Candidate(row[Columns.Token], 1.0)]))%}
+{% endfor %}
+## Vocabulary insights
+
+{% set correct_tokens = set(flat_test.correct_token) %}
```

Comment:
```suggestion
{% set correct_tokens = frozenset(flat_test.correct_token) %}
```

As far as I understand `frozenset` is a built-in - need to check

Reply:
Can't find it anywhere, I see only lists and dicts and tuples are supported.

Reply:
Since better to have a frozen one - immutability is always a good idea.